demo/demo.html

```diff
@@ -3,20 +3,24 @@
   <head>
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
-    <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <script src='./temp/sfx-all-plugins-bundle.js'></script>
+    <!-- above tag to be removed when the cdn PR passes with the updated bundle -->
+    <!-- <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script> -->
   </head>
   <body>
-    <sfx-search-box placeholder="search" clearbutton searchbutton></sfx-search-box>
-    <sfx-sayt visible></sfx-sayt>
+    <sfx-search-box id="searchbox1" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="searchbox1"></sfx-sayt>
+    <sfx-search-box id="searchbox2" placeholder="search" clearbutton searchbutton></sfx-search-box>
+    <sfx-sayt searchbox="searchbox2" minSearchLength=5></sfx-sayt>
```

Comment:
HTML attributes should be all lowercase. Also, even if they're numbers, attribute values should still have quotes around them.

```suggestion
    <sfx-sayt searchbox="searchbox2" minsearchlength="5"></sfx-sayt>
```

Reply:
@DanielMoniz-GBI I originally had it camelcase here, and camelcase in the sayt.ts file that you commented on. @jmbtutor requested this be changed to all lowercase as you can see above, and having it as `min-search-length` in the html and receiving it as camel case `minSearchLength` in sayt.ts was not working? @geoffcoutts maybe something to 100% solve when we pair

Reply:
nvm^